akka-actor-tests/src/test/scala/akka/event/EventStreamSpec.scala

```diff
@@ -73,25 +73,18 @@ class EventStreamSpec extends AkkaSpec(EventStreamSpec.config) {
 
   "An EventStream" must {
 
-    "manage subscriptions" taggedAs GHExcludeTest in {
+    "manage subscriptions" in {
       //#event-bus-start-unsubscriber-scala
       val bus = new EventStream(system, true)
       bus.startUnsubscriber()
       //#event-bus-start-unsubscriber-scala
 
       bus.subscribe(testActor, classOf[M])
-      // subscription is async
-      var n = 41
-      awaitAssert {
-        n += 1
-        bus.publish(M(n))
-        expectMsg(M(n))
-      }
-      bus.unsubscribe(testActor)
-      // unsubscription is async
-      awaitAssert {
-        n += 1
-        bus.publish(M(n))
+      bus.publish(M(42))
+      within(1.second) {
```

Comment:
why is this so tight?

Reply:
I think we can increase it, I just reverted back to the original logic here.